docs/specs/signatures/description.markdown

```diff
@@ -0,0 +1,383 @@
+      BIP: ?
```

Comment:
Shouldn’t we use our rfc format instead of this?

Reply:
Maybe this should be an RFC? I wrote it for the people at Ledger as documentation, wanted to stick it in our repo as docs somehow. Do you think the RFC directory is the right home for this documentation?

Reply:
Hm, I think I like separating spec from RFC. RFC in theory does more explain+alternatives exploration. Part of the consensus-making process to arrive at a spec.